content/_deployments.md

```diff
@@ -351,3 +351,129 @@ The visibility value must be `tenant` because global visibility is not allowed.
 
 ### Response
 A `Deployment` resource.
+
+
+## Update Deployment
+
+> Request Example
+
+```shell
+$ curl -X PUT \
+    -H "Content-Type: application/json" \
+    -H "Tenant: <manager-tenant>" \
+    -u <manager-username>:<manager-password> \
+    -d '{"skip_install": "<skip_install>", "skip_uninstall": "<skip_uninstall>", "skip_reinstall": "<skip_reinstall>", "force": "<force>", "ignore_failure": "<ignore_failure>", "install_first": "<install_first>", "blueprint_id": "<blueprint_id>", "inputs": "<inputs>", "reinstall_list": "<reinstall_list>"}' \
+    "http://<manager-ip>/api/v3.1/deployment-updates/<deployment-id>/update/initiate"
+```
+
+```python
+# Python Client
+client.deployment_updates.update_with_existing_blueprint(skip_install="<skip_install>", skip_uninstall="<skip_uninstall>", skip_reinstall="<skip_reinstall>", force="<force>", ignore_failure="<ignore_failure>", install_first="<install_first>", blueprint_id="<blueprint_id>", inputs="<inputs>", reinstall_list="<reinstall_list>")
+```
+
+> Response Example
+
+```json
+{
+  TODO - DeploymentUpdate object response
+}
+
+```
+
+`PUT "<manager-ip>/api/v3.1/deployment-updates/<deployment-id>/update/initiate"`
+
+Update the deployment. **Supported for Cloudify Manager 4.4 and above.**
+
+### URI Parameters
+* `deployment-id`: The id of the deployment to update.
+
+### Request Body
+
+Property | Type | Description
+--------- | ------- | -----------
+`visibility` | string | Defines who can see the deployment. (Required)
+`skip_install` | boolean | Determines whether to skip install workflow
+`skip_reinstall` | boolean | Determines whether to skip reinstall for nodes whos properties changed
```

Comment:
Determines whether to reinstall the node instances whose properties are modified in the deployment update.